src/components/shared/thread/ActivityGraphFills.js

```diff
@@ -421,16 +421,22 @@ export class ActivityFillGraphQuerier {
     const deviceX = Math.round(cssX * devicePixelRatio);
     const deviceY = Math.round(cssY * devicePixelRatio);
     const categoryUnderMouse = this._categoryAtDevicePixel(deviceX, deviceY);
+
+    const candidateSamples = this._getSamplesAtTime(time);
```

Comment:
I'm not super happy that we now do this heavy task even when hovering on top of some empty space, but I don't have a better idea at the moment, so let's do that.

Reply:
Usually it's super fast, but indeed it's an additonal task that we do now. We can try to optimize it if it becomes a problem.